CONTRIBUTING.md

```diff
@@ -14,26 +14,27 @@ Found a security issue? Please disclose it responsibly. We have a team of develo
 
 ## Contributing to translations
 
-We use [WebTranslateIt][webtranslateit] to manage translation. Please read [our wiki page][translation-wiki] to know more about it. Except if your language isn't on webtranslateit, please do **not** open a pull request to update translations.
+We use [WebTranslateIt][webtranslateit] to manage translations of the app interface. Please read [our wiki page][translation-wiki] to find out more about this. If your language is on WebTranslateIt, please do **not** open a pull request to update translations. If you want to contribute to a language that is not featured on WebTranslateIt, please contact our developers to find out how to proceed.
```

Comment:
We currently have to tell contributors (who want to add a new language) what to do because the wiki page is outdated. Instead of telling people come on IRC, hoping that someone who knows will be available, we should write his answer on the wiki.